src/Markdown.MAML/Transformer/ModelTransformerVersion2.cs

```diff
@@ -133,10 +137,24 @@ protected MamlSyntax SyntaxEntryRule()
                     return null;
                 }
 
-                syntax = new MamlSyntax()
+                var RegExPattern = string.Format("{0}$", MarkdownStrings.DefaultParameterSetModifier.Replace("(","\\(").Replace(")","\\)"));
```

Comment:
Regex is slow and less convenient in this case then `sting.EndsWith`

This change should not be more then 3 lines, please tighten it up